include/seastar/http/common.hh

```diff
@@ -58,6 +58,16 @@ public:
         return path(key);
     }
 
+    std::unordered_map<sstring, sstring>::const_iterator
```

Comment:
nit: if you find it annoying to repeat the type of underlying container here, you could use `auto` or `decltype` or just add a `using container_t = ...` declaration at the top of the class.

Probably not worth changing if nobody else flagged it. Also the seastar guys will have their own readability preferences upstream 🤷

Reply:
Nice, thanks. I think this is fine for now, I'll see what the seastar guys say and adapt it to their liking.